packages/client/helpers/functional-test/run-tests.ts

```diff
@@ -108,6 +108,8 @@ async function main(): Promise<number | void> {
   // If the flag is not provided we want to ignore `relationMode` tests
   if (!args['--relation-mode-tests-only']) {
     jestArgs.push('--testPathIgnorePatterns', 'relationMode-in-separate-gh-action')
+  } else {
+    jestArgs.push('--runInBand')
   }
```

Comment:
```suggestion
  if (args['--relation-mode-tests-only']) {
    jestArgs.push('--runInBand')
  } else {
    jestArgs.push('--testPathIgnorePatterns', 'relationMode-in-separate-gh-action')
  }
```

No double negatives in else, a little bit easier to read.